runtime/lua/vim/_meta/vvars.lua

```diff
@@ -54,13 +54,26 @@ vim.v.cmdbang = ...
 --- @type string
 vim.v.collate = ...
 
+--- @class vim.v.completed_item
+--- @field word? string the text that will be inserted, mandatory
+--- @field abbr? string abbreviation of "word"; when not empty it is used in the menu instead of "word"
+--- @field menu? string extra text for the popup menu, displayed after "word" or "abbr"
+--- @field info? string more information about the item, can be displayed in a preview window
+--- @field kind? string single letter indicating the type of completion
+--- @field icase? integer when non-zero case is to be ignored when comparing items to be equal; when omitted zero is used, thus items that only differ in case are added
+--- @field equal? integer when non-zero, always treat this item to be equal when comparing. Which means, "equal=1" disables filtering of this item.
+--- @field dup? integer when non-zero this match will be added even when an item with the same word is already present.
+--- @field empty? integer when non-zero this match will be added even when it is an empty string
+--- @field user_data? any custom data which is associated with the item and available in |v:completed_item|; it can be any type; defaults to an empty string
+--- @field abbr_hlgroup? string an additional highlight group whose attributes are combined with |hl-PmenuSel| and |hl-Pmenu| or |hl-PmenuMatchSel| and |hl-PmenuMatch| highlight attributes in the popup menu to apply cterm and gui properties (with higher priority) like strikethrough to the completion items abbreviation
+--- @field kind_hlgroup? string an additional highlight group specifically for setting the highlight attributes of the completion kind. When this field is present, it will override the |hl-PmenuKind| highlight group, allowing for the customization of ctermfg and guifg properties for the completion kind
```

Comment:
Can you render it so the description is above the field with a linebreak for padding.

Something like:
```suggestion
---
--- an additional highlight group specifically for setting the highlight attributes 
--- of the completion kind. When this field is present, it will override the |hl-PmenuKind| 
--- highlight group, allowing for the customization of ctermfg and guifg properties --- for the completion kind
--- @field kind_hlgroup? string
```